### PR TITLE
removes authlib and associated code; removes Flask-WTF from requireme…

### PR DESCRIPTION
**Pull request by lprivette**
…nts.txt

Removes the Authlib library and any associated code. I also removed the Flask-WTF from the requirements.txt as it was no longer user. Verified application performed as expected after these changes.

- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by lprivette** [comment]
@mbucknell I noticed a unit test failing and am wondering if it's related to the authlib removal. It is in the test_views.py file (https://github.com/NWQMC/WQP_UI/blob/master/server/wqp/portal_ui_blueprint/tests/test_views.py) "TestWqpDownloadProxy" and I think it's doing a dummy post with dummy authorization. Do you have any insights into this test? can it be removed?

**Comment by mbucknell** [comment]
@lprivette , that test can be removed. It was testing the @authentication_required_when_configured decorator

**Comment by lprivette** [comment]
Ok to remove the whole "tests" folder? It just has an empty __init__.py file and that test file

https://github.com/NWQMC/WQP_UI/tree/master/server/wqp/portal_ui_blueprint/tests

**Comment by mbucknell** [comment]
Yes. the tests were added for the autentication and since that is gone there are no tests for the views.py module.

File: server/wqp/portal_ui_blueprint/templates/theme_base.html [hunk 1]
```diff
@@ -48,21 +48,6 @@
...
-                    ProjectMonitoringLocationWeighting: '{{ url_for("portal_ui.wqp_download_proxy", op="ProjectMonitoringLocationWeighting") }}',
-                    Result: '{{ url_for("portal_ui.wqp_download_proxy", op="Result") }}',
-                    Activity: '{{ url_for("portal_ui.wqp_download_proxy", op="Activity") }}',
-                    ActivityMetric: '{{ url_for("portal_ui.wqp_download_proxy", op="ActivityMetric") }}',
-                    ResultDetectionQuantitationLimit: '{{ url_for("portal_ui.wqp_download_proxy", op="ResultDetectionQuantitationLimit") }}',
-                    BiologicalHabitatMetric: '{{ url_for("portal_ui.wqp_download_proxy", op="BiologicalMetric") }}',
-                    Organization: '{{ url_for("portal_ui.wqp_download_proxy", op="Organization") }}'
-                };
-            } else {
-                Config.DOWNLOAD_URLS = Config.QUERY_URLS;
```
Comment (mbucknell) [inline server/wqp/portal_ui_blueprint/templates/theme_base.html:64]: It' hard for me to check this but make sure QUERY_URLS is used in the javascript and not DOWNLOAD_URLS

**Review by mbucknell — APPROVED** [review]

File: server/wqp/portal_ui_blueprint/templates/theme_base.html (hunk 1 above)
Comment (lprivette) [reply server/wqp/portal_ui_blueprint/templates/theme_base.html:64]: One function in the queryService.js is using the DOWNLOAD_URLS. the getFormUrl https://github.com/NWQMC/WQP_UI/blob/a6a4f90119f09b09e1699dda917f03f0ceb9c3b1/assets/js/queryService.js#L89

I will switch that to QUERY_URLS

**Pull request merged**